app/src/main/java/io/appium/uiautomator2/core/UiAutomatorBridge.java

```diff
@@ -49,26 +50,29 @@ public UiAutomatorBridge() {
         } catch (Error error) {
             Log.e("ERROR", "error", error);
             throw error;
+        } catch (UiAutomator2Exception error) {
+            Log.e("ERROR", "error", error);
```

Comment:
Elsewhere in this PR you've changed `Log.e` to `Log.error`. Not sure of the reason for that, but does it apply here too?